src/docs/faq.md

```diff
@@ -103,17 +103,9 @@ Please read this page before asking for help. Your help request will be pointed
 
 ## Linux Questions
 
-### How come audio isn't working on Ubuntu / Ubuntu forks?
+### Audio isn't working on Debian, Ubuntu or some other Distribution?
 
-- Ubuntu and Ubuntu-based distros may have issues and are not supported, try switching to another distro. Debian is a great alternative if you are used to Ubuntu.
-
-### Why are Ubuntu/Ubuntu based distros not supported?
-
-- They consistently break packages.
-- They have started to force snap on their users.
-- They have published an LTS distro with a non-LTS kernel.
-
-Becuse of these things, Ubuntu is unsupported. We will not help you fix issues on Ubuntu.
+- Old (>1 year) LTS releases may have issues and are not supported, try switching to another distro.
```

Comment:
Here you say "are not supported". But below you say "may not be supported".

Reply:
"may have issues and are not supported" is same on both